Assets/MixedRealityToolkit-Examples/StandardShader/Scenes/MaterialGallery.unity

```diff
@@ -0,0 +1,5793 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Ideally, the scene also needs to follow the updated standards.   The InputManager should be in a folder named "Managers" in the scene hierarchy.

Reply:
I'll make this change too. I just used the Mixed Reality Toolkit->Apply Mixed Reality Scene Settings menu item to set up the input manger. So, we should probably open up an issue to make sure that menu item conforms to the new standard.

Reply:
Please open the issue and link it here. =)

Reply:
> Ideally, the scene also needs to follow the updated standards.

Did we make that a standard?  We really need to document these more.
Also, what if in my projects I want that object named `Globals` ? 😝

Reply:
Added an issue! #1768